client/src/components/Login/loginForm/index.jsx

```diff
@@ -0,0 +1,48 @@
+import React, { Component } from 'react';
+import image from './user.png';
+import { Icon } from 'antd';
+import './style.css'
+
+export default class Login extends Component {
+  state = {
+    username: '',
+    password: '',
+    loginError: false,
+  }
+
+  handleChange = ({ target: { name, value } }) => {
+    this.setState({ [name]: value });
+  }
+
+  login = (event) => {
+    event.preventDefault();
+    const { password, username } = this.state;
+    if (!password.trim() || !username.trim()) {
+      this.setState({ loginError: true });
+    }
+    //fetch//
+  }
+
+  render() {
+    return (
+      <div className='login__form__container'>
+        <img className='login__image' src={image} alt='user' />
+        <form onSubmit={this.login} className='login__form'>
+          <div className='login__input-container'>
+            <Icon className='login__form__icon' type="user" />
+            <input onChange={this.handleChange} placeholder='Username' className='login__form__input' id='username' name='username' type='text' value={this.state.username}></input>
```

Comment:
I think the `input` is a self-closing tag, so it doesn't take any children
so you don't need to call it like this `<input></input>`